src/dropin-model.js

```diff
@@ -50,13 +63,30 @@ EventEmitter.createChild(DropinModel);
 
 DropinModel.prototype.initialize = function () {
   var self = this;
+  var dependencyReadyInterval = setInterval(function () {
+    var ready = true;
+
+    Object.keys(self.dependencyStates).forEach(function (dep) {
+      if (self.dependencyStates[dep] === dependencySetupStates.INITIALIZING) {
+        ready = false;
+      }
+    });
+
+    if (!ready) {
+      return;
+    }
```

Comment:
We may be able to condense this whole block down to:

```
if(Object.values(self.dependencyStates).includes(dependencySetupStates.INITIALIZING)) {
    return;
}
```

Reply:
Yeah, that looks much better, but we can't do it currently because IE 11 does not support `Object.values`.

Definitely something to look into for the next major version when we'll be transpiling from Typescript anyway.

Reply:
Micro-optimization: If you use a for-var-in-loop here rather than the forEach, you can return early as soon as INITIALIZING is seen, rather than looping through every dependency every time or even using a ready-var. As far as condensed-ness goes, it's more condensed than the current code.